internal/host/validator.go

```diff
@@ -182,7 +182,7 @@ func (v *validator) printHasMinCpuCores(c *validationContext, status validationS
 	case ValidationSuccess:
 		return "Sufficient CPU cores"
 	case ValidationFailure:
-		return fmt.Sprintf("Require at least %d CPU cores, found only %d", v.hwValidatorCfg.MinCPUCores, c.inventory.CPU.Count)
+		return fmt.Sprintf("The host is not eligible to participate in Openshift Cluster because the minimum required CPU cores for any role is %d, found only %d", v.hwValidatorCfg.MinCPUCores, c.inventory.CPU.Count)
```

Comment:
"does not", or "do not"?

Reply:
I waffled back and forth on this 🙂 and apparently for good reason, because you're right - this message should actually be `do not` because the subject - `# CPU cores` - is plural. In the other message `does not` is correct because `# GiB of RAM` is singular. [Reference 1](https://english.stackexchange.com/questions/268385/usage-of-do-not-vs-does-not), [2](https://english.stackexchange.com/questions/457242/why-does-this-sentence-use-does-not-instead-of-do-not). Updated, thanks!